File: Java/Mahasiswa.java
/* Saya Hakasa Putri mengerjakan Latihan 2 DPBO 2023 C2 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */

public class Mahasiswa{

    //atribut private kelas Mahasiswa
    private String NIM = "" ;
    private String Nama = "";
	private String Gender = "";
	private String Fakultas = "";
	private String Prodi = "";

    
    public Mahasiswa(){ //konstruktor kosong

    }

    //metode getter dan setter
	

	public String getNIM() {
		return this.NIM;
	}

	public void setNIM(String NIM) {
		this.NIM = NIM;
	}

	public String getNama() {
		return this.Nama;
	}

	public void setNama(String Nama) {
		this.Nama = Nama;
	}

	public String getGender() {
		return this.Gender;
	}

	public void setGender(String Gender) {
		this.Gender = Gender;
	}

	public String getFakultas() {
		return this.Fakultas;
	}

	public void setFakultas(String Fakultas) {
		this.Fakultas = Fakultas;
	}

	public String getProdi() {
		return this.Prodi;
	}

	public void setProdi(String Prodi) {
		this.Prodi = Prodi;
	}

    //untuk print atribut kelas Mahasiswa
    public void getInfoMahasiswa(){

        System.out.println("NIM    		 : " + getNIM());
        System.out.println("Nama         : " + getNama());
		System.out.println("Gender       : " + getGender());
		System.out.println("Fakultas     : " + getFakultas());
		System.out.println("Prodi        : " + getProdi());
        
    } 

}

File: PHP/Program/Mahasiswa.php
<?php

/* Saya Hakasa Putri mengerjakan Latihan 2 DPBO 2023 C2 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */

include "SivitasAkademik.php"; //import class Sivitas Akademik

class Mahasiswa extends SivitasAkademik{ //class Mahasiswa turunan dari class Sivitas Akademik

    //atribut private kelas mahasiswa

    private $fakultas;
    private $prodi;
    private $nim;

    public function __construct(){ //constructor

    }

    //setter getter

	public function getfakultas() {
		return $this->fakultas;
	}

	public function setfakultas($fakultas) {
		$this->fakultas = $fakultas;
	}

	public function getprodi() {
		return $this->prodi;
	}

	public function setprodi($prodi) {
		$this->prodi = $prodi;
	}

    public function getnim() {
		return $this->nim;
	}

	public function setnim($nim) {
		$this->nim = $nim;
	}

    //untuk print atribut kelas Mahasiswa
    public function getInfoMahasiswa(){

        echo "NIM               : ". $this->getnim()."<br/>";
		echo "Fakultas          : ". $this->getfakultas()."<br/>";
        echo "Prodi             : ". $this->getprodi()."<br/>";
    } 

	public function __destruct(){ //destructor

	}
}
